lib/active_model/serializer.rb

```diff
@@ -6,6 +6,7 @@
 require 'active_model/serializer/configuration'
 require 'active_model/serializer/fieldset'
 require 'active_model/serializer/lint'
+require 'active_model/serializer/logging'
```

Comment:
this should be required in lib/active_model_serializers, I think. It's more of a global concern than local to this file. feel free to disagree